packages/components/src/components/expansion-panel/expansion-panel.jsx

```diff
@@ -69,31 +63,21 @@ class ExpansionPanel extends React.Component {
         );
     };
 
-    render() {
-        const { message } = this.props;
-
-        return (
-            <>
-                <div
-                    className={classNames('dc-expansion-panel__header-container', {
-                        'dc-expansion-panel__header-active': this.state.is_open,
-                    })}
-                >
-                    {message.header}
-                    <Icon
-                        icon='IcChevronDownBold'
-                        className='dc-expansion-panel__header-chevron-icon'
-                        onClick={this.onClick}
-                    />
-                </div>
-                {!!this.state.is_open &&
-                    (Array.isArray(message.content)
-                        ? this.renderArrayContent(message.content.slice())
-                        : message.content)}
-            </>
-        );
-    }
-}
+    return (
+        <>
+            <div
+                className={classNames('dc-expansion-panel__header-container', {
+                    'dc-expansion-panel__header-active': is_open,
+                })}
+            >
+                {message.header}
+                <Icon icon='IcChevronDownBold' className='dc-expansion-panel__header-chevron-icon' onClick={onClick} />
+            </div>
+            {!!is_open &&
```

Comment:
No need to cast here.

```suggestion
            {is_open &&
```

Reply:
True, I applied this